manifests/init.pp

```diff
@@ -102,8 +105,9 @@
   Stdlib::Port $api_port = 24817,
   Stdlib::Host $content_host = '127.0.0.1',
   Stdlib::Port $content_port = 24816,
-  Stdlib::Absolutepath $webserver_static_dir = '/var/lib/pulp/docroot',
-  Stdlib::Absolutepath $pulp_static_root = '/var/lib/pulp/assets',
+  Stdlib::Absolutepath $apache_docroot = $user_home,
+  Stdlib::Absolutepath $pulpcore_media_root = $user_home,
+  Stdlib::Absolutepath $pulpcore_static_root = "${user_home}/assets",
```

Comment:
I ran into this with my Apache PR as well. While it's the same, I think this would be clearer in the intent. It's because the assets need to be served to the end user.
```suggestion
  Stdlib::Absolutepath $pulpcore_static_root = "${apache_docroot}/assets",
```